Misc/Regions_GB Landmass Medium Detail.txt

```diff
@@ -1,4 +1,4 @@
-REGIONNAME A-SouthWest 
```

Comment:
Happy with these changes to remove whitespace, but I think this is a creep over from #3390